### DISCUSSION TITLE
Dark Font Color in Dark Mode

### DISCUSSION THREAD

**Discussion by simon-innovamaxx** [breezefront/community#26] [1. Help]
In the Breeze Blank Theme, the font color of the menu items is not readable if the system settings are set to dark mode.
Maybe someone could fix that.

https://share.getcloudapp.com/wbuQGZYK

**Comment by vovayatsyuk** [comment]
Thanks for reporting. Fixed and will be included in a large 2.0 update that we will release this or next week.
